crates/fuel-core/src/service/sub_services.rs

```diff
@@ -160,14 +178,21 @@ pub fn init_sub_services(
     #[cfg(not(feature = "p2p"))]
     let p2p_adapter = P2PAdapter::new();
 
-    let gas_price_provider = StaticGasPrice::new(config.static_gas_price);
+    let old_gas_price_provider = StaticGasPrice::new(config.static_gas_price);
```

Comment:
It seems we should be able to remove this=)

Reply:
Yes. The new provider isn't implemented for the tx pool and graphql ports yet.